distribution/docs/src/main/resources/content/_running/ingest-command.adoc

```diff
@@ -12,6 +12,7 @@ The ${command-console} has a command-line option for ingesting data.
 [NOTE]
 ====
 Ingesting with the console ingest command creates a metacard in the catalog, but does not copy the resource to the content store.
+The Ingest Command requires read access to the directory being ingested from, see the <<{architecture-prefix}url_resource_reader, URL Resource Reader>> for configuring read permissions on the directory.
```

Comment:
Remove the preposition "from". The phrase "to the directory being ingested" reads better without it.

Reply:
Run-on sentence. Break it into two sentences here:

> ...to the directory being ingested. See the <<...

Reply:
agree, updated